.github/workflows/on-push.yml

```diff
@@ -55,7 +55,3 @@ jobs:
         uses: docker/build-push-action@v2.7.0
         with:
           file: Dockerfile
-      - name: Build and push Docker image
```

Comment:
@mboudreau remind me why we deleted this? Seems like a lifetime ago that we talked about it.

Reply:
Because in the Dockerfile.prod, we will try to install the analytics package from github, which simply won't work because it doesn't have access to PAC and will break the build because of it.

I think this was needed as we'll have the same issue with Brahmos and since we're now have a decent CD system going, the feedback loop should be good enough (I think).  It would be nice to have BB run when we have a PR to make sure the dockerfile.prod build works as expected...

Reply:
It would wouldn't it? :laughcry: